cpp/src/arrow/compute/kernels/aggregate_test.cc

```diff
@@ -3693,6 +3693,11 @@ TEST_F(TestSkewKurtosis, Options) {
     AssertSkewKurtosisInvalid(type, {"[]", "[]", "[]"}, options);
     AssertSkewKurtosisAre(type, "[0, 1, null, 2]", options, 0.0, -1.5);
     AssertSkewKurtosisAre(type, {"[0, 1]", "[]", "[null, 2]"}, options, 0.0, -1.5);
+    options.bias = false;
+    AssertSkewKurtosisAre(type, {"[0, 1]", "[]", "[null, 2]"}, options, 0.0, NAN);
```

Comment:
Are we sure we want to return NAN, or rather null since the situation is similar to `min_count`?

Reply:
I am checking there are enough samples for Skew and Kurtosis same as we do with `min_count` both on `aggregate` and `hash_aggregate` , I suppose this is what you meant?